go/pkg/client/cas_test.go

```diff
@@ -124,11 +124,12 @@ func TestRead(t *testing.T) {
 	defer c.Close()
 
 	tests := []struct {
-		name   string
-		fake   fakes.Reader
-		offset int64
-		limit  int64
-		want   []byte // If nil, fake.blob is expected by default.
+		name                   string
+		fake                   fakes.Reader
+		offset                 int64
+		limit                  int64
+		want                   []byte // If nil, fake.blob is expected by default.
+		disableCompressionTest bool
```

Comment:
you're not using this disableCompressionTest, right?
Although I'd rather add a bool compressed here, and set it explicitly in some cases.

Reply:
I was going to use it for the `limit` test cases below, then I realized I could do it differently -- removed now.

I also technically prefer using a "compressed" here, but I think we do can have a pragmatic check (that is, check whether limit is set) instead.